deploy/namespace.yaml

```diff
@@ -4,3 +4,6 @@ metadata:
   name: $NAMESPACE
   labels:
     name: $NAMESPACE
+    pod-security.kubernetes.io/enforce: privileged
```

Comment:
why do we need this here ?

if pod-security labels are not added then pod-security mechanism will not kick in for the namespace.
is my understanding wrong ?

Reply:
I belive your understanding is wrong. The default behavior depends on how PSA has been configured, hence we can't assume privileged by default.

Reply:
OK, i see the built in PSA admission controller [1] has default config, i assume for when these labels are not present on namespace.

[1] https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-admission-controller/

Reply:
Yes, the default is configurable and that's why we need to be explicit with what permissions we require